cogstat/cogstat.py

```diff
@@ -642,6 +666,7 @@ def filter_outlier(self, var_names=None, mode='2.5mad'):
                     text_output += _('The following cases will be excluded: ')
                     text_output += cs_stat._format_html_table(excluded_cases.to_html(bold_rows=False,
                                                                                      classes="table_cs_pd"))
+                    # TODO convert Mahalanobis distance cut-off to variable scale
```

Comment:
I'm not sure this is possible/makes sense on a single dimension. Probably, in this case, lower and upper limit cannot be displayed, but we display only the excluded cases according to each variable.

Reply:
Agreed. Chart is now not displayed in case of 'mahalanobis'.